src/main/resources/templates/dev-hub/developerGuidelines.html

```diff
@@ -11,61 +11,99 @@
+    <p class="govuk-body">A majority of the resources returned by the Companies House API contain
+      members that reference enumeration types. This helps make the resources self-documenting, and
+      allows clients to interpret the meaning of a resource member without needing to parse a text
+      description.
+    </p>
+    <p class="govuk-body">
+      Enumeration types are used to supplement or replace a text description, which allows clients
+      to display their own version of a description or provide descriptions in multiple languages.
+    </p>
+    <p class="govuk-body">
+      The collection of enumeration types used by Companies House are available on <a
+        href="https://github.com/companieshouse/api-enumerations">GitHub</a>. These files provide
+      mapping between enumeration type and text description, and are divided
+      into sets or classes. Each API resource member will define which class of enumeration is being
+      returned.
+    </p>
+    <p class="govuk-body">
+      A planned enhancement to the enumeration scheme is the provision of API endpoints that will
+      return the enumeration class catalogue. This avoids enumerations having to be hard-coded
+      within a client, and by periodically checking for change through ETags, clients do not have to
+      download the full catalogue.
+    </p>
+  </section>
   <section id="resources">
     <h2 class="govuk-heading-m">Data resources</h2>
-    <p class="govuk-body">Data is mostly returned as JSON documents. Your application must tolerate
-      the
-      order of document members changing over time, and expect to receive members it hasn't seen
-      before.</p>
+    <p class="govuk-body">
+      Data is mostly returned as JSON documents. Your application must be able to handle the order
+      of document members changing over time and expect to receive members it has not seen before.
+    </p>
   </section>
-  <section id="security">
-    <h2 class="govuk-heading-m">Security</h2>
-    <p class="govuk-body">The API can only be accessed over TLS. We recommend using TLS 1.2.</p>
+  <section id="appSecurity">
+    <h2 class="govuk-heading-m">Application security</h2>
+    <p class="govuk-body">The API can only be accessed over Transport Layer Security (TLS). We
+      recommend using TLS 1.2.</p>
   </section>
-  <section id="keysecurity">
-    <h2 class="govuk-heading-m">Securing API keys</h2>
-    <p class="govuk-body">It is important to keep your API keys secure, to prevent them being
-      discovered, your
-      account being compromised and your rate-limit quota being exceeded. There are a few best
-      practices that can help with this:</p>
+  <section id="keySecurity">
+    <h2 class="govuk-heading-m">API key security </h2>
+    <p class="govuk-body">It is important to keep your API keys secure. This will prevent them from
+      being discovered, your account from being compromised and your rate-limit quota from being
+      exceeded. </p>
```

Comment:
Paragraphs seem to have a space character before they are closed. Please clean that up.